docs/source/markdown/podman-run.1.md.in

```diff
@@ -258,6 +258,43 @@ Invalid if using **--dns**, **--dns-option**, or **--dns-search** with **--netwo
 
 If used together with **--pod**, the container will not join the pod's network namespace.
 
+- **pasta[:OPTIONS,...]**: use **pasta**(1) to create a user-mode networking
+stack. By default, IPv4 and IPv6 addresses and routes, as well as the pod
+interface name, are copied from the host. If port forwarding isn't configured,
+ports will be forwarded dynamically as services are bound on either side (init
+namespace or container namespace). Port forwarding preserves the original source
+IP address. Options described in pasta(1) can be specified as comma-separated
+arguments.
+
+
+In terms of pasta(1) options, **--config-net** is given by default, in order to
+configure networking when the container is started. Also, **-t none** and
+**-u none** are passed if, respectively, no TCP or UDP port forwarding from host
+to container is configured, to disable automatic port forwarding based on bound
+ports. Similarly, **-T none** and **-U none** are also given to disable the same
+functionality from container to host.
+
+Some examples:
+  - **pasta:--no-map-gw**: Don't allow the container to directly reach the host
+    using the gateway address, which would normally be mapped to a loopback or
+    link-local address.
+  - **pasta:--mtu,1500**: Specify a 1500 bytes MTU for the _tap_ interface in
+    the container.
+  - **pasta:--ipv4-only,-a,10.0.2.0,-n,24,-g,10.0.2.2,--dns-forward,10.0.2.3,-m,1500,--no-ndp,--no-dhcpv6,--no-dhcp**,
+    equivalent to default slirp4netns(1) options: disable IPv6, assign
+    `10.0.2.0/24` to the `tap0` interface in the container, with gateway
+    `10.0.2.3`, enable DNS forwarder reachable at `10.0.2.3`, set MTU to 1500
+    bytes, disable NDP, DHCPv6 and DHCP support.
+  - **pasta:--no-map-gw,-I,tap0,--ipv4-only,-a,10.0.2.0,-n,24,-g,10.0.2.2,--dns-forward,10.0.2.3,--no-ndp,--no-dhcpv6,--no-dhcp**,
+    equivalent to default slirp4netns(1) options with Podman overrides: same as
+    above, but leave the MTU to 65520 bytes, and don't map the gateway address
+    from the container to a local address.
+  - **pasta:-t,auto,-u,auto,-T,auto,-U,auto**: enable automatic port forwarding
+    based on observed bound ports from both host and container sides
+  - **pasta:-T,5201**: enable forwarding of TCP port 5201 from container to
+    host, using the loopback interface instead of the tap interface for improved
+    performance
+
```

Comment:
Please remove. You already added it to `options/network.md`. This is a showstopper, I'm afraid, so I have to request changes.

Reply:
Agreed, this warrants a re-spin indeed, sorry for that.

By the way, if you're curious, it came from an out-of-tree version I had around for way too long, which I happened to write before you reworked man pages including `network.md`.

Reply:
Done, removed.